build.gradle

```diff
@@ -64,7 +64,7 @@ dependencies {
     implementation 'commons-io:commons-io:2.6'
 
     implementation 'com.squareup.retrofit2:retrofit:2.5.0'
-    implementation 'com.squareup.okhttp3:okhttp:3.12.3'
+    implementation 'com.squareup.okhttp3:okhttp:4.0.1'
 
```

Comment:
@burningthumb Can you revert this commit? It was decided to stay on legacy 3.12.* for now. See discussion [here](https://github.com/nextcloud/Android-SingleSignOn/commit/e781a33e0115ab8c3f4983fed2def1ab924fb9d9#diff-c197962302397baf3a4cc36463dce5ea)